python/cudf/cudf/core/series.py

```diff
@@ -695,7 +695,7 @@ def append(self, to_append, ignore_index=False, verify_integrity=False):
         return super()._append(to_append, ignore_index, verify_integrity)
 
     @_cudf_nvtx_annotate
-    def reindex(self, index=None, copy=True):
+    def reindex(self, *args, **kwargs):
```

Comment:
We also need to update `Parameters` with `fill_value` in kwargs? Should we be explicit that `copy` and `fill_value` are `kwargs` only?

Reply:
I updated this docstring, normalizing it to pandas and adding this parameter.